From: =?utf-8?q?Bastien_Roucari=C3=A8s?= <rouca@debian.org>
Date: Fri, 6 Sep 2019 12:34:24 +0200
Subject: Depends on lastest unicode

Depends on newly virtual package node-unicode-lastest

bug-debian; https://bugs.debian.org/934079
Forwarded: not-needed
---
 bin/generate-identifier-regex.js | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/bin/generate-identifier-regex.js b/bin/generate-identifier-regex.js
index d661d72..eea22f3 100644
--- a/bin/generate-identifier-regex.js
+++ b/bin/generate-identifier-regex.js
@@ -1,13 +1,15 @@
 'use strict';
 
 // Which Unicode version should be used?
+/*
 let pkg = require('../package.json')
 let dependencies = Object.keys(pkg.devDependencies)
 let unicodeVersion = dependencies.find((name) => /^unicode-\d/.test(name))
+*/
 
-let start = require(unicodeVersion + '/Binary_Property/ID_Start/code-points.js').filter(ch => ch > 0x7f)
+let start = require('unicode-lastest' + '/Binary_Property/ID_Start/code-points.js').filter(ch => ch > 0x7f)
 let last = -1
-let cont = [0x200c, 0x200d].concat(require(unicodeVersion + '/Binary_Property/ID_Continue/code-points.js')
+let cont = [0x200c, 0x200d].concat(require('unicode-lastest' + '/Binary_Property/ID_Continue/code-points.js')
     .filter(ch => ch > 0x7f && search(start, ch, last + 1) === -1))
 
 function search(arr, ch, starting) {
